app/workers/end_of_cycle/send_application_deadline_has_passed_email_to_candidates_worker.rb

```diff
@@ -0,0 +1,33 @@
+module EndOfCycle
+  class SendApplicationDeadlineHasPassedEmailToCandidatesWorker
+    include Sidekiq::Worker
+
+    BATCH_SIZE = 120
+
+    def perform
+      return unless EmailTimetable.send_application_deadline_has_passed_email_to_candidates?
+
+      BatchDelivery.new(relation:, batch_size: BATCH_SIZE).each do |batch_time, application_forms|
+        SendApplicationDeadlineHasPassedEmailToCandidatesBatchWorker.perform_at(batch_time, application_forms.pluck(:id))
+      end
+    end
+
+    def relation
+      ApplicationForm
```

Comment:
I'm not sure if it's possible but should we use a `distinct` here incase we have duplicate Candidates being returned 🤔

Reply:
Smart!